########################################
# Messages
########################################
float32 range_min
float32 range_max
float32 range_front
float32 range_front_min
float32 range_back
float32 range_back_min
float32 angle_range_min
float32 angle_range_max
float32 angle_front_min
float32 angle_back_min
